### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by Nyrroz** [codewars/codewars.com#2874] [Q&A] [answered]
Hello, yes. I'm at fault. My curiosity got the best of me and I wanted to know what would happen if you simply copied a random response from GitHub for a 1kyu test.
I'm sorry for "cheating," I really been enjoying codewars for a while now (almost 3 years) and I had completed over 350 katas and was ranked 4kyu.  

Please let me have full access to my account back, even if that means a complete reset of my account and loosing all my stats. 
I am motivated to work my way up again because at the end of the day I am addicted to this site. 

Thank you for reading and have a nice day !

**Comment by Blind4Basics** [comment]
Hello,

...And what's your account, exactly?

**Comment by Blind4Basics** [comment, marked as answer]
well, considering that you copied:
  * 8x 1 kyu (all)
  * 5x 2 kyu (all) 
  * 3x 3kyu (all)
  * as least half of the 4 kyu you solved

The answer to your request is no. You can delete your current account and start fresh if you want.
Side note: you completed 115 kata, _including the copies_, not 350 (_why_ ppl are always lying about everything??)

Cheers

**Reply by Blind4Basics** [reply]
Ah, yeah, that's the honor you ~~gained~~ got-left-after-invalidations by completing kata.
If you really wanna have a clean account, the way to go is to delete it. You won't lose much (less than 80 kata. Some users have thousands of them). By deleting your account, you can create a new one with the same name/infos.

I'm definitely no planing on going through all your current kata and lose time to decide if this or that should or not be invalidated, sorry. Too many of them.

**Reply by Nyrroz** [reply]
What if you invalidate all of them ?
Please I am begging you, I would rather loose all my progress and start all over again as long as it is on my account. 
I am really attached to this account since I created it in 2019. 
Codewars is what got me to like the concept of solving problems and the competitive side of it motivates me. 
I don't know what to tell you more that this. Truly thank you for your time.

**Reply by Kacarott** [reply]
> and the competitive side of it motivates me.

Don't you think that a competitive site should ban people who cheat around the rules? Letting cheaters stay is unfair to the people who are motivated by the competitive side but are actually honest.

**Reply by Nyrroz** [reply]
I think that you are absolutely right. I never cheated for the 3 years I have been on codewars. I have to say that curiosity got the best of me and I wanted to know if it was possible to get 1kyu that way. 
I understand my mistake and I know I deserve to be punished for it.  
I have been banned for multiple weeks now I think. I just want to make up for it. 
That's why I want to repay my debt by loosing all my progress up until now. 
I would also love to contribute any way I can.

**Reply by Blind4Basics** [reply]
Are you on CW's discord? Can you get there? If so, DM me (-> `B4B` ; nickname:`B1³B`, yellow mod)
